### PR TITLE
data: aaronmack

### PR DESCRIPTION
**Pull request by aaronmack**

### CODE REVIEW

**Review by accodes21 — DISMISSED** [review]

**Review by harshsingh32 — DISMISSED** [review]

**Review by khushalarora — APPROVED** [review]
Thank you for contributing :tada:

File: data/aaronmack.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Aaron Li",
+  "type": "personal",
+  "displayStatusPublic": true,
```
Comment (SaraJaoude) [inline data/aaronmack.json:4]: this is no longer needed:
```suggestion
```

File: data/aaronmack.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "avatar" "https://github.com/aaronmack.png",
```
Comment (SaraJaoude) [inline data/aaronmack.json:5]: this is no longer needed
```suggestion
```

File: data/aaronmack.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "url": "https://xyzzyxwz.top:5173"
+    },
+    {
+      "title": "Finder",
+      "date": "April 2023",
+      "icon": "FaExternalLinkSquareAlt",
+      "color": "grey",
+      "description": "To explore (python-nicegui)",
+      "url": "https://server1.xyzzyxwz.top:12030"
+    },
```
Comment (SaraJaoude) [inline data/aaronmack.json:49]:
```suggestion
    }
```

**Pull request merged**